apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -170,7 +176,7 @@ export class AddJob {
     filterVariables: IFilterVariables,
     delayAmount: number | undefined
   ) {
-    command.bridgeResponse = await this.fetchResonateData(command, filterVariables);
+    await this.fetchResonateData(command, filterVariables);
```

Comment:
Do we need to use the response somewhere here?

Reply:
Not here we use it in `fetchResonateData` and update by ref the job, see below 
```
command.job.digest = { ...command.job.digest, ...jobDigest } as IWorkflowStepMetadata;
```

I am unsure what the most convenient way of doing it will be.